parlai/agents/seq2seq/seq2seq.py

```diff
@@ -17,10 +17,17 @@
 
 
 class Seq2seqAgent(Agent):
-    """Simple agent which uses an LSTM to process incoming text observations."""
+    """Simple agent which uses an RNN to process incoming text observations.
```

Comment:
it looks like a lot of changes but they're almost all aesthetic / refactoring / commenting. only real change is that I now support loading models which had different params (the current params are overridden, with a messaged displayed for each)